src/python/pants/backend/jvm/tasks/jvm_compile/rsc/rsc_compile.py

```diff
@@ -863,6 +811,7 @@ def _collect_metai_classpath(self, metacp_result, relative_input_paths):
         os.path.join(relative_workdir, 'resolve', 'ivy', '[^/]*', 'ivy', 'jars', '.*'),
```

Comment:
This fix no longer includes adding `coursier`, which I think it did originally? It will need to use the new location of the coursier cache.

Reply:
Oh, right